### DISCUSSION TITLE
ipxe bootstrap

### DISCUSSION THREAD

**Discussion by stelb** [kairos-io/kairos#392] [General]
Hi,

i tried to install kairos with ipxe (more specific: netbootxyz/netboot.xyz)
I took the provided ipxe file as a start, but it bails out with:
"unable to mount root fs on unknown-block(0, 0)"

kernel and initrd is successfully loaded, is livenet include in initrd to support root=live:http://.. ?

Regards

Stefan

**Comment by mudler** [comment]
Thanks for bringing this up, This looks rather an issue as it should work - I'll create an issue out of it

**Comment by mudler** [comment]
Can you try to repack the squashfs file with gzip? it defaults with xz and that might cause compatibility issues. I'm currently switching to gz now as default, check out here for a follow up. https://github.com/kairos-io/kairos/issues/393
